infinum_json_api_setup.gemspec

```diff
@@ -20,11 +20,11 @@ Gem::Specification.new do |s|
 
   s.add_development_dependency 'jsonapi-query_builder'
   s.add_development_dependency 'jsonapi-serializer'
+  s.add_development_dependency 'overcommit', '~> 0.58'
```

Comment:
I just saw that json schemer is locked to a very specific version 🆙 . Can we relax that while we're here? An update of the json schemer gem would require a new release of this gem to unblock all projects that want to update it.

Reply:
@djordjelacmanovic what do you say? You were the one who[ added json-scemer](https://github.com/infinum/infinum-json-api-setup/pull/19/files#diff-86838e23024bcd6d3920071974141a2adac1c608c48eda305c4ca56edf8a2f17R16)

Reply:
Sure, we can relax it. But shouldn't this setup also be kept up to date with json-schemer anyway?

Reply:
It should, how does relaxing version constraint prevent us in that?

Reply:
It doesn't, just makes one update it here first before doing it in their own project, that's all I meant. Basically just ensuring this gets some TLC

Reply:
On second thought, since we will invest time and make sure to keep gems up to date the 'defensive' approach to achieve is not the best way to get it so the reasoning above doesn't stand.

Reply:
🤘 tnx